api/src/UserManager/helpers/PasswordHelper.ts

```diff
@@ -0,0 +1,11 @@
+import * as bcrypt from "bcrypt";
+import {ILoginForm} from "../LoginRegister/Test/fetchInput.test";
```

Comment:
relocate this inside a DTO, its still an interface but used as DTO and named as such

Reply:
DTO is a class or interface ?

Reply:
Here DTO can be an interface and used as a class, thanks to how interfaces work in TS